pkg/builder/build.go

```diff
@@ -148,6 +148,18 @@ func (blder *Builder) Build(r reconcile.Reconciler) (manager.Manager, error) {
 		return nil, err
 	}
 
+	if blder.mgr.GetWebhookServer() != nil {
+		svr := blder.mgr.GetWebhookServer()
+		err := svr.EnableDefaultsFor(blder.mgr.GetScheme(), blder.apiType)
+		if err != nil {
+			return nil, err
+		}
+		err = svr.EnableValidationFor(blder.mgr.GetScheme(), blder.apiType)
```

Comment:
Won't `svr` have access to the scheme ? If yes, then we can get rid of scheme parameter from `EnableDefaultsFor` and `EnableValidationFor`.

Reply:
`svr` won't be able to have the Scheme until the inject time.
But the setup time is earlier than the inject time.

Another option is having 2 rounds of injection:
1st round injects deps for the server
2nd round inject deps to the handlers under the server.

It's not clear to me which way is better.